### ISSUE TITLE
Turnilo should be more friendly for kubernetes/mesos (liveness vs. readiness)

### ISSUE THREAD

**Issue by mkuthan** [allegro/turnilo#338]
Current healthcheck is great, it checks that underlaying Druid broker is able to serve the queries. This is a readiness check, the application itself is ok but temporarily is not able to serve reports due to problems with Druid broker(s). We need second check - liveness - if the application is not alive kubernetes/mesos should try to restart the application. See: 
* https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/
* https://console.bluemix.net/docs/java-spring/healthcheck.html

In summary:
- existing healtcheck should be used as readiness check 
- new liveness check should return success if the application is running (no additional checks, as lightweight as possible)

I would rename healthEndpoint to readinessEndpoint and add new livenessEndpoint. Configuration option healthEndpoint could be an alias for readinessEndpoint (to keep backward compatibility and to allow smooth migration to the new version).
